From dca1a100d14056865c7360d80a2a1d1ae4b0de96 Mon Sep 17 00:00:00 2001
From: Gerd Hoffmann <kraxel@redhat.com>
Date: Fri, 10 Feb 2023 10:49:44 +0100
Subject: [PATCH 5/5] rh openssl: add crypto/bn/rsa_sup_mul.c to file list

RH-Author: Gerd Hoffmann <kraxel@redhat.com>
RH-MergeRequest: 27: openssl update
RH-Bugzilla: 2164534 2164550 2164565 2164583
RH-Acked-by: Miroslav Rezanina <mrezanin@redhat.com>
RH-Commit: [2/2] a097fc031b54208e9c8974173784e4c306dbf7a0 (kraxel/centos-edk2)
---
 CryptoPkg/Library/OpensslLib/OpensslLib.inf       | 1 +
 CryptoPkg/Library/OpensslLib/OpensslLibCrypto.inf | 1 +
 2 files changed, 2 insertions(+)

diff --git a/CryptoPkg/Library/OpensslLib/OpensslLib.inf b/CryptoPkg/Library/OpensslLib/OpensslLib.inf
index e446b51e66..7e78255467 100644
--- a/CryptoPkg/Library/OpensslLib/OpensslLib.inf
+++ b/CryptoPkg/Library/OpensslLib/OpensslLib.inf
@@ -576,6 +576,7 @@
   $(OPENSSL_PATH)/ssl/statem/statem_local.h
 # Autogenerated files list ends here
 # RHEL8-specific OpenSSL file list starts here
+  $(OPENSSL_PATH)/crypto/bn/rsa_sup_mul.c
   $(OPENSSL_PATH)/crypto/evp/kdf_lib.c
   $(OPENSSL_PATH)/crypto/evp/pkey_kdf.c
   $(OPENSSL_PATH)/crypto/kdf/kbkdf.c
diff --git a/CryptoPkg/Library/OpensslLib/OpensslLibCrypto.inf b/CryptoPkg/Library/OpensslLib/OpensslLibCrypto.inf
index c207dc8f4c..1c551cb099 100644
--- a/CryptoPkg/Library/OpensslLib/OpensslLibCrypto.inf
+++ b/CryptoPkg/Library/OpensslLib/OpensslLibCrypto.inf
@@ -526,6 +526,7 @@
   $(OPENSSL_PATH)/crypto/x509v3/v3_admis.h
 # Autogenerated files list ends here
 # RHEL8-specific OpenSSL file list starts here
+  $(OPENSSL_PATH)/crypto/bn/rsa_sup_mul.c
   $(OPENSSL_PATH)/crypto/evp/kdf_lib.c
   $(OPENSSL_PATH)/crypto/evp/pkey_kdf.c
   $(OPENSSL_PATH)/crypto/kdf/kbkdf.c
-- 
2.31.1

